Mathlib/Data/Real/EReal.lean

```diff
@@ -909,30 +956,37 @@ theorem neg_strictAnti : StrictAnti (- · : EReal → EReal) :=
 protected theorem neg_le {a b : EReal} : -a ≤ b ↔ -b ≤ a := by
  rw [← neg_le_neg_iff, neg_neg]
 
-/-- if `-a ≤ b` then `-b ≤ a` on `EReal`. -/
+/-- `-a ≤ b → -b ≤ a` on `EReal`. -/
```

Comment:
I must say that I prefer the previous version with "if..."

Reply:
I don't have a strong opinion about this, I mainly changed it for consistency, since `neg_le` already had the comment:
'''
/-- `-a ≤ b ↔ -b ≤ a` on `EReal`. -/
'''

Do you think I should also change the `↔` to `iff` there?

Reply:
Yes, personally I find it nicer that we use plain normal English in comments everywehere.